recipe/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 0
   skip: true  # [win32 or osx]
```

Comment:
Adding `skip: true  # [py==27]` will skip the Python 2.7 builds on all platforms.  This is likely easier than modifying the `conda-forge.yml`file.

Reply:
Doesn't work, still can't re-render :(